profiler/test/Datadog.Profiler.Native.Tests/ManagedThreadListTest.cpp

```diff
@@ -29,15 +29,15 @@ TEST(ManagedThreadListTest, CheckRemove)
     ASSERT_EQ(threads.Count(), 3);
 
     bool found = false;
-    ManagedThreadInfo* pInfo = nullptr;
+    std::shared_ptr<ManagedThreadInfo> pInfo = nullptr;
```

Comment:
Is it the same as calling reset()?

Reply:
No, we just initialize the variable to nullptr.